src/main/resources/db/migration/V17__imrt_ddl.sql

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************
+* IMRT-416: Adds TTS provided/required fields to item table
+********************************************************************/
+ALTER TABLE item ADD COLUMN sight_tts_provided VARCHAR NOT NULL DEFAULT '';
+ALTER TABLE item ADD COLUMN visual_tts_provided VARCHAR NOT NULL DEFAULT '';
+ALTER TABLE item ADD COLUMN visual_tts_required VARCHAR NOT NULL DEFAULT '';
+
+-- Populate new fields with ingested item.json data
```

Comment:
I guess we can do this but I prefer to use our migration tool rather than set default values.